vendor/bat-native-ads/src/bat/ads/internal/history/category_content_value_util.cc

```diff
@@ -31,16 +31,17 @@ base::Value::Dict CategoryContentToValue(
 CategoryContentInfo CategoryContentFromValue(const base::Value::Dict& root) {
   CategoryContentInfo category_content;
 
-  if (const auto* value = root.FindString(kCategoryKey)) {
-    category_content.category = *value;
+  if (const auto* category_key_value = root.FindString(kCategoryKey)) {
```

Comment:
same, do we need the verbosity? Thanks

Reply:
Fixed in b4174716b278ed73d7aab2c1ed841d54e25cb51a.